### ISSUE TITLE
Remove Deactivate from YellowCube Sync-Flag Action

### ISSUE THREAD

**Issue by hersches** [swisspost-yellowcube/magento-yellowcube#14]
In ../Swisspost/YellowCube/Model/Observer.php

Since we can't reactivate a "Delete"-Flaged Article in the YellowCube-Warehouse-Management-System, we shouldn't allow to deactivate an Article only changeing the Sync-Flag. The *   - From Yes to No = deactivate from YC is" is not ok.

        /**
	         * Scenario
	         *
	         * - product is disabled or enabled => no change
	         * - yc_sync_with_yellowcube is Yes/No
	         *   - From No to Yes => insert into YC
	         *   - From No to No => no change
	         ***   - From Yes to No = deactivate from YC**
@psavary 
Lines 183-186 sould be removed from Observer.php:

        if (!(bool)$product->getData('yc_sync_with_yellowcube') && $this->hasDataChangedFor($product, array('yc_sync_with_yellowcube'))) {
	            $this->getSynchronizer()->deactivate($product);
	            return $this;
	        }

**Comment by psavary** [comment]
Meaning, once it is activated, it cannot be deactivated? Is deactivation the same as deletion for YC?

**Comment by hersches** [comment]
No, once it is deactivated, it cannot reactivated by the merchant. So if for instance for a time-period, the article should be not sychronized and fulfilled by YellowCube, there is no way to change this status back. For now, the code shouldn't generate any action if the status change locally to 'No'.
For the future, we have define probably a explicit action-button to transmit "deactivate completely" to us with the deactivate function. 
To know: There is no common database behind and the article in fact exist during live-time because of 'revision tracks in audits'. Deactivate-Flag is set and stays for ever.
